### PR TITLE
[feature]:: Mariadb plugin

### LINKED ISSUE #3953: MariaDB integration
### Is your proposal related to a problem?

I saw MariaDB integration in your roadmap. I've already done most of the work related to this and need a bit more time on testing this out

### Describe the solution you'd like

MariaDB integration

### If the feature is approved, would you be willing to submit a PR?

Yes

### PR DESCRIPTION
**Pull request by vishnurk6247**
Closes #3953

### CODE REVIEW

File: plugins/packages/mariadb/lib/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    if (sourceOptions.ssl_enabled) poolConfig['ssl'] = sslObject;
+
+    const mariadbPool = mariadb.createPool(poolConfig);
+
+    return mariadbPool.getConnection();
+  }
+
+  private toJson(data) {
+    return JSON.parse(
```
Comment (gsmithun4) [inline plugins/packages/mariadb/lib/index.ts:60]: Do we need to do this?
JSON.parse(data) is enough right?

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: plugins/packages/mariadb/lib/index.ts (hunk 1 above)
Comment (vishnurk6247) [reply plugins/packages/mariadb/lib/index.ts:60]: Remember getting a big int error after querying during development. This fixed the issue

**Comment by vishnurk6247** [comment]
Hey @gsmithun4, I've rebased the changes locally and pushed the updated feature branch. Please verify

**Comment by gsmithun4** [comment]
deployed https://tj-mariadb.herokuapp.com/

**Comment by gsmithun4** [comment]
@vishnurk6247 shall we give some default value to `connection limit`, since the connection limit is mandatory.

**Comment by Nandinisaha13** [comment]
Hi @vishnurk6247 there are 2 things I found-

**Comment by vishnurk6247** [comment]
Hey @gsmithun4, will make the change update the feature branch

**Comment by vishnurk6247** [comment]
Hey @Nandinisaha13, MariaDB has three different logos, the one there is in the screenshot you uploaded and another two which are black and white in color. The one you see over there is a neutral color and would be visible in both modes(though it's relatively less visible in dark mode). Also if we were planning to switch the logos based on the theme, I'm not sure whether Tooljet supports this as of now as there is option to add only one logo file. Could you please check this with the team and suggest some changes regarding this?

**Comment by Nandinisaha13** [comment]
Hy @vishnurk6247 so I have asked the team, if there could be a possible change for the logo, waiting for the reply, in the meanwhile can you change the name to "MariaDB" ?

**Comment by vishnurk6247** [comment]
Hey @Nandinisaha13, sure

**Review by gsmithun4 — APPROVED** [review]
LGTM

**Comment by withshubh** [comment]
Hey @vishnurk6247 - Can you share your twitter handle? :)

**Comment by vishnurk6247** [comment]
Hey @withshubh , it's the same '@vishnurk6247'

**Pull request merged**